src/transport_ofi.h

```diff
@@ -1213,4 +1213,63 @@ void shmem_transport_syncmem(void)
     // transport routine will be a nop until an API is provided.
 }
 
+static inline
+uint64_t shmem_transport_get_pending_put_cntr(shmem_transport_ctx_t *ctx)
+{
+    uint64_t cnt = 0;
```

Comment:
No need to initialize this to 0 (potentially adds an instruction since this is stack-allocated, although the compiler should see that this is not read before it's written below).